astroid/rebuilder.py

```diff
@@ -119,10 +120,26 @@ def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
                 ):
                     doc = first_value.value if PY38_PLUS else first_value.s
                     node.body = node.body[1:]
-                    return node, doc
+                    if PY38_PLUS:
+                        doc_node = nodes.Const(
+                            value=doc,
+                            lineno=first_value.lineno,
+                            col_offset=first_value.col_offset,
+                            parent=node,
+                            end_lineno=node.end_lineno,
```

Comment:
Shouldn't this be `first_value`? Looking at the test this does not seem correct. The docstring only spans one line, but `lineno` and `end_lineno` are not the same.